conmon/conmon.c

```diff
@@ -1329,6 +1331,21 @@ int main(int argc, char *argv[])
 		if (dup2(slavefd_stderr, STDERR_FILENO) < 0)
 			pexit("Failed to dup over stderr");
 
+		/* If LISTEN_PID env is set, we need to set the LISTEN_PID
+		   it to the new child process */
+		char *listenpid = getenv("LISTEN_PID");
+		if (listenpid != NULL) {
+			int lpid = atoi(listenpid);
```

Comment:
`atoi` has no error checking (`strtol` does).  Do we care if the old `LISTEN_PID` value was a non-integer?  Or <= 0?  I'm fine with ignoring them, but wanted to make sure it was an intentional choice.